Banyan/src/sessions.jl

```diff
@@ -309,7 +313,7 @@ function wait_for_session(session_id::SessionId=get_session_id(), kwargs...)
     elseif session_status == "completed"
         error("Session with ID $session_id has already completed")
     elseif session_status == "failed"
-        error("Session with ID $session_id has failed")
+        error("Session with ID $session_id has failed. See above for logs.")
```

Comment:
We should only say see above if this session is returning logs to client side... right? Otherwise, it should say to check website.